src/sharedFramework/sharedFramework.proj

```diff
@@ -123,8 +123,7 @@
       <SharedFrameworkDepsFile>$(SharedFrameworkNameAndVersionRoot)\$(SharedFrameworkName).deps.json</SharedFrameworkDepsFile>
       <RuntimeGraphGeneratorRuntime Condition="'$(OSGroup)'=='Windows_NT'">win</RuntimeGraphGeneratorRuntime>
       <RuntimeGraphGeneratorRuntime Condition="'$(OSGroup)'=='OSX'">osx</RuntimeGraphGeneratorRuntime>
-      <RuntimeGraphGeneratorRuntime Condition="$(OutputRID.StartsWith('alpine'))">alpine</RuntimeGraphGeneratorRuntime>
-      <RuntimeGraphGeneratorRuntime Condition="'$(RuntimeGraphGeneratorRuntime)'==''">linux</RuntimeGraphGeneratorRuntime>      
+      <RuntimeGraphGeneratorRuntime Condition="'$(RuntimeGraphGeneratorRuntime)'==''">linux</RuntimeGraphGeneratorRuntime>
```

Comment:
Do we really want the whole `linux` runtime graph in the shared fx .deps.json file?  It probably won't cause issues, but it is bloat/unnecessary.

The way this works is that it reads all the `runtime.json` files in all the packages, and scans all the RIDs looking for RIDs that import this specified RID.  So in this case, all the RIDs that import `linux` will get put in the .deps.json file, like `ubuntu`, `rhel`, etc.  However, since we are building for `linux-musl`, it seems like we should only be including RIDs that import `linux-musl`.

Reply:
I looked at this and personally felt it was probably OK to just include the entire runtime graph. Is there any harm beyond a size?

Reply:
I don't *think* there is any harm, but following that logic we should put the whole graph in all the time - put Windows RIDs in for Linux, Linux in for Windows, etc.